frontend/templates/about-page.html

```diff
@@ -47,10 +47,13 @@
         <a href="/about-page">About</a>
         <a href="/contact-us">Contact Us</a>
     <a href="/our-team">Contact List</a>
-        <p>Event Registration:</p>
-        <a href="/event/smtf">SMTF</a>
-        <a href="/event/irshad">Irshad</a>
-        <a href="/event/inverse">InVerse</a>      
+        <a href="/sponsors">Sponsors</a>
+
+    <p>Event Registration:</p>
+    <a href="/event/smtf">SMTF</a>
+    <a href="/event/irshad">Irshad</a>
+    <a href="/event/inverse">InVerse</a>
+    <a href="/event/csr">CSR</a>      
```

Comment:
Move this below Sponsors as CSR is not an event in which participant can register.